### PR TITLE
Feat/mr 13

### PR DESCRIPTION
**Pull request by Arzoid29**
adding the why sales assist process section

### CODE REVIEW

File: src/components/atoms/instantInfo/index.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      <div className='grid grid-cols-3 gap-10 w-full text-orange-normal font-semibold'>
+        <Rates rate='+23%' title='Closing Rate' />
+        <Rates rate='15hrs+' title='Saved Per Person Per Month' />
+        <Rates rate='+15%' title='Positive Reviews' />
+      </div>
+    </div>
+  )
+}
+
+
```
Comment (brya0x) [inline src/components/atoms/instantInfo/index.tsx]: **Change**: this is not an atom is too big to be an atom is more a molecule and i gave you some changes on another pr

File: src/components/organism/sections/home/instantChanges/index.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+             </div>
+             <div className=' hidden lg:flex'>
+             <InstantChangesDesktop />
+             </div>
+          </div>
+        </div>
+      </Section>
+    </section>
+  )
+}
```
Comment (brya0x) [inline src/components/organism/sections/home/instantChanges/index.tsx]: use function no const

File: src/components/organism/sections/home/whySalesAssistSales/index.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            key={business.code}
+            code={business.code}
+            name={business.name}
+            kind='div'
+          />
+        ))}
+      </div>
+    </section>
+  );
+};
```
Comment (brya0x) [inline src/components/organism/sections/home/whySalesAssistSales/index.tsx]: use function on const

File: src/data/sales-data/icon.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+ */
+export function buildIcon({ data, code, fallback, }: BuilderIcon) {
+    const current = data.find(d => d.code === code);
+
+    if (current) {
+        return <Image width={124} height={254} alt={current.code} src={current.path}  />
+    }
+
+    return fallback
+}
```
Comment (brya0x) [inline src/data/sales-data/icon.tsx]: **Breaking Change**: this a componente

File: src/data/sales-data/industries.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+    return returnKind(
+        <>
+            <div className ="flex  items-center  justify-center rounded-full bg-gray-200 ">
+                {buildIcon({ data: industries, code: code, fallback: <></>})}
+            </div>
+            <p className='mt-2 text-gray-300 '>{name}</p>
+        </>
+    )
+}
```
Comment (brya0x) [inline src/data/sales-data/industries.tsx]: **Breaking change**: this is a component

**Review by code-bryan — CHANGES_REQUESTED** [review]

File: src/components/atoms/leading/index.tsx [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+
+  export default function Leading({ title, content }: LeadingProps) {
+    return (
+      <div>
+        <h3 className='font-semibold text-[1.5rem] leading-6'>{title}</h3>
+        <span className='flex justify-center leading-6 text-gray-400 font-medium text-[1.3rem]'>{content}</span>
+      </div>
+    );
+  }
```
Comment (brya0x) [inline src/components/atoms/leading/index.tsx]: **Change**: be careful with identation

**Review by code-bryan — CHANGES_REQUESTED** [review]

**Review by code-bryan — APPROVED** [review]

**Pull request merged**